ds3-autogen-c/src/main/resources/templates/request-templates/AmazonS3GetRequest.ftl

```diff
@@ -0,0 +1,29 @@
+<#-- ********************************** -->
+<#-- Generate "AmazonS3 Get Request" -->
+<#--   Input: Source object             -->
+<#-- ********************************** -->
+<#list getRequests() as requestEntry>
+    <#if (requestEntry.getClassification().toString() == "spectras3") && (requestEntry.getVerb().toString() == "GET")>
```

Comment:
The comment at the top says AmazonS3 Get Requests, but this check here is for spectras3.  Is this correct?  Also, same comment about filtering out in the generator before code generation.